app/models/models.py

```diff
@@ -1,6 +1,7 @@
-from pydantic import BaseModel
 from typing import Union
 
+from pydantic import BaseModel
```

Comment:
t comes after p -- this isn't sorted?

Reply:
it sorts, but after empty line, basically the new "import group" is sorted on its own. It's a bit similar like in Go